### ISSUE TITLE
[Feature Request] iCloud Syncing

### ISSUE THREAD

**Issue by Catta1997** [SuperMarcus/NineAnimator#113]
**Is your feature request related to a problem? Please describe.**
Yes, I’m using an iPad and my iPhone to watch anime but I have to do every time “export history” to sync the device

**Describe the solution you'd like**
iCloud syncing so if I watch an anime on my iPad it shows as “completed” on other device

**Comment by uttiya10** [comment]
Why don't you use MyAnimeList to keep track of your watch history?

**Comment by SuperMarcus** [comment]
MAL and other tracking services doesn't store any episode progresses.

**Comment by uttiya10** [comment]
Yes, your right. Only NineAnimator keeps track of episode prograss.

**Comment by Valzion** [comment]
Would this be able to be used soon? or is this just on the 'to-do-list'
